src/patterns/addresses/multiple/index.njk

```diff
@@ -41,6 +41,16 @@ stylesheets:
     classes: "govuk-!-width-two-thirds",
     id: "address-town",
     name: "address-town",
+    autocomplete: "address-level1"
```

Comment:
It looks like we're bumping town or city to `address-level1` and using `address-level2` for county. This is a change from [the version of this pattern before the changes in 72f058611525678584f279c194812a278d69c95f](https://github.com/terrysimpson99/govuk-design-system/blob/67fdfbbd8d7d5dcae5873312561af35c9e0b03fe/src/patterns/addresses/multiple/index.njk), so we need to be sure it's correct. If anything, I'd expect it be the other way around based on the descriptions in the HTML spec, where level4 is the _finest grained_ and level 1 is the 'province etc'.

It looks like there was previously some discussion around this in https://github.com/alphagov/govuk-design-system/issues/1090, and in https://github.com/alphagov/govuk-design-system/pull/1100#issue-537025646 we explicitly opted _not_ to add an autocomplete attribute for county.

If we're not 100% confident in this change, my preference would be to revert to the autocomplete attributes we were using prior to removing the county field.

If we are doing this intentionally, I think we need to clearly document this change and its rationale as part of the commit message or PR description (ideally both).